ddtrace/config.py

```diff
@@ -0,0 +1,14 @@
+from .pin import Pin
+
+
+def get_from(obj):
+    """Retrieves the configuration for the given object.
+    Any object that has an attached `Pin` must have a configuration
+    and if a wrong object is given, an empty `dict` is returned
+    for safety reasons.
+    """
+    pin = Pin.get_from(obj)
+    if pin is None:
+        return {}
```

Comment:
Maybe I should `log.debug()` something here? the reason why we return a new dictionary is to avoid crashing users application. On the other hand, we fail silently.

Reply:
Yes worth a `log.debug`